profile/src/main/java/rdx/works/profile/data/model/apppreferences/Security.kt

```diff
@@ -1,19 +1,82 @@
+@file:OptIn(ExperimentalSerializationApi::class)
+
 package rdx.works.profile.data.model.apppreferences
 
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.EncodeDefault
+import kotlinx.serialization.ExperimentalSerializationApi
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 import rdx.works.profile.data.model.Profile
+import rdx.works.profile.data.model.factorsources.FactorSource
+import rdx.works.profile.data.model.factorsources.FactorSourceKind
+import java.time.Instant
 
 @Serializable
 data class Security(
     @SerialName("isDeveloperModeEnabled")
     val isDeveloperModeEnabled: Boolean,
     @SerialName("structureConfigurationReferences")
-    val structureConfigurationReferences: List<Boolean>,
+    val structureConfigurationReferences: List<StructureConfigurationReference>,
```

Comment:
just an example how to access the factorSourceKind of the structureConfigurationReference
```
fun test() {
    val structureConfigurationReference = Security.StructureConfigurationReference(
        metadata = Security.StructureConfigurationReference.Metadata(),
        configuration = Security.StructureConfigurationReference.Configuration(
            numberOfDaysUntilAutoConfirmation = 1,
            primaryRole = mapOf(),
            recoveryRole = mapOf(),
            confirmationRole = mapOf()
        ),
    )
    
    val factorSourceKindOfPrimaryRole = structureConfigurationReference
        .configuration
        .primaryRole[Security.StructureConfigurationReference.Configuration.Role.PrimaryRoleTag]
        ?.kind
}
```